.github/workflows/pages.yml

```diff
@@ -25,6 +25,7 @@ jobs:
 
   build:
     runs-on: ubuntu-latest
+    timeout-minutes: 3
```

Comment:
Why is the timeout needed?

Reply:
b.c. the build seems to hang indefinitely when installing `react-scripts`